### PR TITLE
Fix: Send reminder modal ui and options UI

### PR DESCRIPTION
**Pull request by mayankagnihotri7**
## Designs
- [Send Reminder Modal](https://www.figma.com/file/4gdPDRPqGqQqPYsO3Qj8CG/Miru-App?type=design&node-id=14889%3A208433&mode=design&t=3AaVPFSt6iL2nPbU-1)
- [Options on invoice details page](https://www.figma.com/file/4gdPDRPqGqQqPYsO3Qj8CG/Miru-App?type=design&node-id=12636%3A207136&mode=design&t=3AaVPFSt6iL2nPbU-1)

## Description
- Fixed send reminder modal UI for the labels.
- Fixed the overlapping issue where when the options were opened it had text overlapping it.

## Preview
Before -
Screenshot 2023-07-06 at 3 45 27 PM

Screenshot 2023-07-06 at 3 46 02 PM

After -
Screenshot 2023-07-06 at 3 43 53 PM

Screenshot 2023-07-06 at 3 44 18 PM

### CODE REVIEW

**Comment by apoorv1316** [comment]
@mayankagnihotri7 Few fixes are required here
1. Remove the '{}' from the "Send Invoice" heading.
2. Email  & Message label is not similar to designs.
3. Email value background should have more margin from the bottom of the box
4. Subject label text is small
Screenshot 2023-07-06 at 4 36 42 PM

File: app/javascript/src/components/Invoices/List/SendInvoice/index.tsx [hunk 1]
```diff
@@ -152,62 +152,75 @@ const SendInvoice: React.FC<any> = ({
...
+                ))}
+                <label
+                  className="form__label relative bottom-4 right-44 z-1"
+                  htmlFor="to"
+                >
+                  Email
+                </label>
+                {/* <input
+                    name="to"
+                    ref={input}
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/SendInvoice/index.tsx:181]: Why is this code commented out? If it's not used anymore please remove it.

File: app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx [hunk 2]
```diff
@@ -133,69 +133,59 @@ const SendInvoice: React.FC<any> = ({
...
+                <Recipient
+                  email={recipient}
+                  handleClick={() => handleRemove(recipient)}
+                  key={recipient}
+                />
+              ))}
+              {/* <input
                     name="to"
                     ref={input}
                     style={{ width }}
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx:191]: ditto

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/components/Invoices/List/SendInvoice/index.tsx (hunk 1 above)
Comment (mayankagnihotri7) [reply app/javascript/src/components/Invoices/List/SendInvoice/index.tsx:181]: It was commented out before I even used modal.

File: app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx (hunk 2 above)
Comment (mayankagnihotri7) [reply app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx:191]: It was commented out before I even used modal.

File: app/javascript/src/components/Invoices/List/SendInvoice/index.tsx (hunk 1 above)
Comment (prasanthchaduvula) [reply app/javascript/src/components/Invoices/List/SendInvoice/index.tsx:181]: Please remove it if it not used

File: app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx (hunk 2 above)
Comment (prasanthchaduvula) [reply app/javascript/src/components/Invoices/popups/SendInvoice/index.tsx:191]: Please remove it if it not used

**Comment by apoorv1316** [comment]
@mayankagnihotri7 Can we close the menu after the modal is opened?
Screenshot 2023-07-07 at 4 23 46 PM

**Review by apoorv1316 — APPROVED** [review]

**Comment by mayankagnihotri7** [comment]
@apoorv1316 Yes. We can close the modal using the esc key.

**Comment by apoorv1316** [comment]
Screenshot 2023-07-07 at 6 17 42 PM
@mayankagnihotri7 We are able to type in the email field and add other emails. I don't think we are now allowing that. Please fix it

**Comment by mayankagnihotri7** [comment]
@apoorv1316 I've made the requested changes. Please review.

**Comment by apoorv1316** [comment]
@mayankagnihotri7 We can still type emails on the mobile screens. Please fix it

**Comment by mayankagnihotri7** [comment]
@apoorv1316 I've made the requested changes. Please review.

**Review by apoorv1316 — APPROVED** [review]

**Comment by apoorv1316** [comment]
@prasanthchaduvula Please review.

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**